src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -68,7 +68,7 @@
     <BuildActions>$(BuildActions) $(FlagParameterPrefix)pack</BuildActions>
     <BuildActions>$(BuildActions) $(FlagParameterPrefix)publish</BuildActions>
 
-    <BuildArgs>$(FlagParameterPrefix)ci</BuildArgs>
+    <BuildArgs Condition="'$(ContinuousIntegrationBuild)' == 'true'">$(FlagParameterPrefix)ci</BuildArgs>
```

Comment:
I think we need to have this condition to not disrupt source-build until we change the defaults. Same for the condition in L119.

```suggestion
    <!-- TODO: Remove the DotNetBuildSourceOnly condition when the default build behavior changes to dev -->
    <BuildArgs Condition="'$(ContinuousIntegrationBuild)' == 'true' or '$(DotNetBuildSourceOnly)' == 'true'">$(FlagParameterPrefix)ci</BuildArgs>
```